test/wpt/test-url.js

```diff
@@ -17,13 +17,14 @@ runner.defineGlobal('DOMException', {
     // A 'hack' to get the DOMException constructor since we don't have it
     // on the global object.
     if (DOMException === undefined) {
-      const port = new (require('worker_threads').MessagePort)();
+      const port = new (require('worker_threads').MessageChannel)().port1;
```

Comment:
hmm, not relevant to this PR, but I think we may be better off just exporting DOMException in the worker binding and use that here...

Reply:
I’m marking this as resolved since you’re already implementing that in #28072 :)